tests/integration/test_vulnerability_detector/test_providers/test_update_interval.py

```diff
@@ -71,15 +72,15 @@ def test_update_interval(get_configuration, configure_environment, restart_modul
 
     # Check that the update has not started prematurely
     with pytest.raises(TimeoutError):
-        wazuh_log_monitor.start(timeout=global_parameters.default_timeout,
+        wazuh_log_monitor.start(timeout=VULN_DETECTOR_GLOBAL_TIMEOUT,
                                 callback=callback_provider_database_updating)
         raise AttributeError(f'Unexpected event {provider} database updating')
 
     # Travels the time set in the update interval parameter
     check_time_travel(time_travel=True, interval=timedelta(seconds=seconds_to_travel))
 
-    # Check that the feed is downloaded after the set update interval
+    # Check that the feed is downloaded after the set update interval(10s for vuln thread refresh + 5s to wait callback)
```

Comment:
This comment can be split into two different lines.

Reply:
Done in 77223ed24d8fff611947e817d180bf744720ed4c